### PR TITLE
Fixed lord serve

### LINKED ISSUE #28: Lord serve not working?
I tried running `lord serve --host 0.0.0.0 --port 8080` but I cannot find the dashboard in the allocated location. Is this broken atm? I have run the command in ubuntu and raspberry pi but I see 404 page not found.

### PR DESCRIPTION
**Pull request by samarpan-rai**
Fixes #28

### CODE REVIEW

**Comment by samarpan-rai** [comment]
I also added `time taken (ms)` column as I thought this was handy. Feel free to remove it if you don't like it.

File: skedulord/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-version = "0.2.5"
+version = "0.2.4"
```
Comment (samarpan-rai) [inline skedulord/__init__.py]: The changes to the assets were not being pushed so I messed around a little during debugging so this was changed.

**Comment by koaning** [comment]
image


image

Is it maybe possible to bundle some of these js files together? Something is telling me that this is maybe *too* many changes.

**Comment by samarpan-rai** [comment]
I thought so too but there seems to be no way

Kind Regards,
Samarpan Rai

________________________________

**Comment by samarpan-rai** [comment]
> image
> 
> image
> 
> Is it maybe possible to bundle some of these js files together? Something is telling me that this is maybe _too_ many changes.

The majority of files are coming from `gatsby build` and according to this discussion https://github.com/gatsbyjs/gatsby/issues/6548 it is not possible. I have no to little experience with gatsby so maybe you know a way to squash them all together?

**Comment by samarpan-rai** [comment]
One way we could fix this problem is by adding a [gatsby action](https://github.com/marketplace/actions/github-action-for-gatsby-cli) to build the required artifacts.  However, this would mean that, during development, these artifacts have to be re-built and if there is some version difference then we may have unexpected behaviors.

**Comment by koaning** [comment]
I am closing this PR. It's my fault for ignoring this repo for so long, sorry about that, but there are new features on the roadmap. 

One of them is to use `rich` as a rendering engine for the front-end. It will be a big overhaul that I'm happy to do. The new version will be way better, I'm also working on a config system for cron so that you no longer need to fiddle around with virtual environments. Unfortunately this PR won't fit the direction of the project anymore so I'm going to close it.

**Pull request closed without merging**